src/coupling/balance.cc

```diff
@@ -474,7 +457,6 @@ void Balance::finish_source_assembly(unsigned int quantity_idx)
 	chkerr(MatAssemblyEnd(region_source_matrix_[quantity_idx], MAT_FINAL_ASSEMBLY));
 	chkerr(MatAssemblyBegin(region_source_rhs_[quantity_idx], MAT_FINAL_ASSEMBLY));
 	chkerr(MatAssemblyEnd(region_source_rhs_[quantity_idx], MAT_FINAL_ASSEMBLY));
-	chkerr(MatMultTranspose(region_source_rhs_[quantity_idx], ones_, region_source_vec_[quantity_idx]));
```

Comment:
I think ones_ is no more used and can be removed.

Reply:
done, removed also ones_be_.